filetests/isa/x86/windows_fastcall_x64.clif

```diff
@@ -48,8 +48,8 @@ ebb0(v0: f64, v1: f64, v2: f64, v3: f64):
 ; nextln:                         store notrap aligned v8, v7
 ; nextln:                         store notrap aligned v9, v7+16
 ; check:                          v11 = stack_addr.i64 ss0
-; nextln: [Op2fldDisp8#410,%xmm7] v13 = load.f64x2 notrap aligned v11+16
```

Comment:
As I mention in the commit message, I do not know having this part of the check causes an error. I tried copying the style in f.ex `probestack-disabled.clif`, and I swear this passed once, but now yields an error like
```
> [RexOp1spaddr8_id#808d,%rcx]        v11 = stack_addr.i64 ss0               
                                      ^~~~~~~~~~~~~~~~~~~~~~~~            
Matched #7: \bv11 = stack_addr\.i64 ss0\b                                
> [Op2fldDisp8#410,%xmm7]             v13 = load.f64x2 notrap aligned v11+16
Missed #8: \[Op2fldDisp8\#410,%xmm7\] v13 = load\.f64x2 notrap aligned v11\+16\b
```
note that
```
got:      [Op2fldDisp8#410,%xmm7]             v13 = load.f64x2 notrap aligned v11+16
expected: [Op2fldDisp8#410,%xmm7] v13 = load.f64x2 notrap aligned v11+16
```
these are the same aside from whitespace. I tested having the same amount of whitespace just in case, and that didn't influence passing or not. Am I misunderstanding the `nextln` directive?